support/get_libc_client

```diff
@@ -30,6 +30,11 @@ elif [[ "$STACK" == "scalingo-18" ]]; then
 		http://archive.ubuntu.com/ubuntu/pool/universe/u/uw-imap/uw-imap_${dep_version}~dfsg-7.debian.tar.xz
 		http://archive.ubuntu.com/ubuntu/pool/universe/u/uw-imap/uw-imap_${dep_version}~dfsg-7.dsc
 	)
+elif [[ "$STACK" == "scalingo-20" ]]; then
+	dep_urls+=(
```

Comment:
Can you use `https` for all external calls ? For old stacks too ?

Reply:
archive.ubuntu.com is not available via HTTPS:

```
╰─$ wget "https://archive.ubuntu.com/ubuntu/pool/universe/u/uw-imap/uw-imap_${dep_version}~dfsg.orig.tar.gz"
--2021-09-14 12:06:48--  https://archive.ubuntu.com/ubuntu/pool/universe/u/uw-imap/uw-imap_2007f~dfsg.orig.tar.gz
Resolving archive.ubuntu.com (archive.ubuntu.com)... 91.189.88.142, 91.189.88.152, 2001:67c:1360:8001::24, ...
Connecting to archive.ubuntu.com (archive.ubuntu.com)|91.189.88.142|:443... failed: Connection refused.
Connecting to archive.ubuntu.com (archive.ubuntu.com)|91.189.88.152|:443... failed: Connection refused.
Connecting to archive.ubuntu.com (archive.ubuntu.com)|2001:67c:1360:8001::24|:443... failed: Network is unreachable.
Connecting to archive.ubuntu.com (archive.ubuntu.com)|2001:67c:1360:8001::23|:443... failed: Network is unreachable.
```

Reply:
😢 Okay